packages/components/socialFeed/NewsFeed/MapModal/index.tsx

```diff
@@ -0,0 +1,296 @@
+  ActivityIndicator,
+  Platform,
+  ScrollView,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from "react-native";
+
+import chevronLeft from "../../../../../assets/icons/chevron-left.svg";
+import close from "../../../../../assets/icons/close.svg";
+import location from "../../../../../assets/icons/location.svg";
+import {
+  neutral17,
+  neutral77,
+  neutralA3,
+  primaryColor,
+  secondaryColor,
+  transparentColor,
+} from "../../../../utils/style/colors";
+import {
+  fontMedium16,
+  fontSemibold13,
+  fontSemibold16,
+} from "../../../../utils/style/fonts";
+import { layout } from "../../../../utils/style/layout";
+import { BrandText } from "../../../BrandText";
+import { SVG } from "../../../SVG";
+import { PrimaryButton } from "../../../buttons/PrimaryButton";
+import { SecondaryButtonOutline } from "../../../buttons/SecondaryButtonOutline";
+import { TextInputCustom } from "../../../inputs/TextInputCustom";
+import { TextInputOutsideLabel } from "../../../inputs/TextInputOutsideLabel";
+import ModalBase from "../../../modals/ModalBase";
+import { SpacerColumn, SpacerRow } from "../../../spacer";
+
+const MapComponent = Platform.select({
+  native: () => lazy(() => import("./Map")),
+  default: () => lazy(() => import("./Map.web")),
+})();
+
+let typingTimeout: NodeJS.Timeout | undefined;
+
+interface TMapModalProps {
+  visible: boolean;
+  onClose: () => void;
+  handleSubmit?: () => void;
+}
+
+type Tlocation = [number, number];
+
+export const MapModal: React.FC<TMapModalProps> = ({
+  handleSubmit,
+  onClose,
+  visible,
+}) => {
+  const [description, setDescription] = useState("");
+  const [addressPlaceHolder, setAddressPlaceHolder] = useState("Address");
+  const [address, setAddress] = useState("");
+  const [isLoading, setIsLoading] = useState(false);
+  const [results, setResults] = useState<any>([]);
+  const [locationSelected, setLocationSelected] = useState<Tlocation>([
+    48.8566, 2.3522,
+  ]);
+
+  const searchAddress = async (params: string) => {
+    return new Promise((resolve, reject) => {
+      fetch(
+        `https://nominatim.openstreetmap.org/search?q=${params}&format=json`,
+      )
+        .then((response) => response.json()) // Parse the JSON response
+        .then((data) => resolve(data)) // Resolve the promise with the location data
+        .catch((error) => reject(error)); // Reject the promise with the error
+    });
+  };
+
+  useEffect(() => {
+    if (address.trim() === "") {
+      setIsLoading(false);
+      return;
+    }
+    setIsLoading(true);
+    clearTimeout(typingTimeout);
+    setResults([]);
+    typingTimeout = setTimeout(() => {
+      searchAddress(address)
+        .then((searchResults) => {
+          setResults(searchResults);
+          setIsLoading(false);
+        })
+        .catch(() => {
+          setResults([]);
+          setIsLoading(false);
+        });
+    }, 1500);
+    return () => clearTimeout(typingTimeout);
+  }, [address]);
+
+  return (
+    <ModalBase
```

Comment:
You created a specific component `ModalWithoutHeader`, why don't you use it ? Please do

Reply:
Done